### DISCUSSION TITLE
Does PlexTraktSync only count every movie as played once?

### DISCUSSION THREAD

**Discussion by Night-Rocc** [Taxel/PlexTraktSync#1751] [Q&A] [answered]
I noticed Trakt keeps track of how many times you've watched a movie, and I was wondering if PlexTraktSync keeps track of repeated watches as well? Or does it just mark the movie as watched once and if it's watched again, it won't count?

**Comment by glensc** [comment, marked as answer]
there's no one answer. if you use sync command, then it's recorded only as yes/no. if you use watch command, it's able to record from active play more than one play to trakt.

**Comment by glensc** [comment]
choosing answer for user and closing (they seem disappeared)
